app/main.py

```diff
@@ -2,0 +1,24 @@
+import os.path
+import shutil
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    cmd_name = parsed_command[0]
+    src_file = parsed_command[1]
+    dst_file = parsed_command[2]
+
+    src_file_path = os.path.abspath(src_file)
+    dst_file_path = os.path.abspath(dst_file)
```

Comment:
Code Style: You should use `os.path.join` to concatenate parts of the path. This makes your code cross-platform and it will use either `/` or `\` depending on the operating system.

Reply:
function take filename with path, thats why I use abspath